include/mra/tensor/tensorview.h

```diff
@@ -449,6 +461,13 @@ namespace mra {
     template<typename... Dims>
     requires(sizeof...(Dims) < NDIM && (std::is_integral_v<Dims>&&...))
     SCOPE TensorView<T, NDIM-sizeof...(Dims)> operator()(Dims... idxs) const {
+      std::array<size_type, sizeof...(Dims)> indices = {static_cast<size_type>(idxs)...};
+      for (size_type i = 0; i < indices.size(); ++i) {
+        if (indices[i] >= dim(i)) {
+          THROW("TensorView: index out of bounds");
+        }
```

Comment:
Make this an `assert()` so that it is removed in non-debug builds.

Reply:
Why do we leave out this check in non-debug builds?